block-management/block-manager.js

```diff
@@ -140,10 +170,20 @@ export default class BlockManager extends React.Component<PropsType, StateType>
 const styles = StyleSheet.create( {
 	container: {
 		flex: 1,
+		justifyContent: 'flex-start',
 		backgroundColor: '#caa',
 	},
 	list: {
 		flex: 1,
 		backgroundColor: '#ccc',
 	},
+	htmlView: {
+		flex: 1,
+		backgroundColor: '#fff',
+	},
+	switch: {
+		flexDirection: 'row',
+		justifyContent: 'flex-start',
```

Comment:
Nitpicking, it think `flex-start` is the default option for `justifyContent`.

Reply:
Correct! That's leftover code after experimenting (and failed eventually) with having the switch include the text label.